packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/react-native/installsdk.md

```diff
@@ -0,0 +1,53 @@
+### Install npm package
+
+```bash
+$ npm install @okta/okta-react-native --save
+$ react-native link @okta/okta-react-native
+```
+
+### Install iOS OIDC SDK
+
+1. [**CocoaPods**](https://guides.cocoapods.org/using/getting-started.html)
+
+***React Native >= 0.60***: With React Native 0.60 pods are added to podfile automatically. Run `pod install` command to install dependecies:
```

Comment:
Couple of nits - comma before pds..."the" before `pod install`, dependencies typo (was missing the n)

***React Native >= 0.60***: With React Native 0.60, pods are added to the podfile automatically. Run the `pod install` command to install dependencies: